api-inference-community/docker_images/timm/tests/test_api_image_classification.py

```diff
@@ -11,12 +12,14 @@
     "image-classification" not in ALLOWED_TASKS,
     "image-classification not implemented",
 )
+@parameterized_class(
+    [{"model_id": model_id} for model_id in TESTABLE_MODELS["image-classification"]]
+)
 class ImageClassificationTestCase(TestCase):
     def setUp(self):
-        model_id = TESTABLE_MODELS["image-classification"]
         self.old_model_id = os.getenv("MODEL_ID")
         self.old_task = os.getenv("TASK")
-        os.environ["MODEL_ID"] = model_id
+        os.environ["MODEL_ID"] = self.model_id
         os.environ["TASK"] = "image-classification"
         from app.main import app
 
```

Comment:
Could you also add this below please?

```
@classmethod
def setUpClass(cls):
    from app.main import get_pipeline

    get_pipeline.cache_clear()
```

Reply:
As in https://github.com/huggingface/huggingface_hub/blob/main/api-inference-community/docker_images/generic/tests/test_api_feature_extraction.py

Reply:
Tried this yesterday and it didn't work. however, I think I noticed the problem was that the app itself was a bit out of sync with the template. Will check back in on it

Reply:
You might want to make sure https://github.com/huggingface/huggingface_hub/pull/304 is in your branch as well :)

Reply:
done!